app/mailers/precheck_mailer.rb

```diff
@@ -0,0 +1,38 @@
+class PrecheckMailer < ApplicationMailer
+  DEFAULT_EMAIL = 'josh.starcher@cru.org'.freeze
+  DEFAULT_CHANGES_REQUEST_EMAIL = 'josh.starcher@cru.org'.freeze
+
+  def precheck_completed(family)
+    email = if Rails.env.production?
+              family.attendees.pluck(:email).select(&:present?).compact
+            else
+              DEFAULT_EMAIL
+            end
```

Comment:
I see this if statement behaviour is copied from the FamilyMailer, do you think it could make sense to extract it to a method?